### ISSUE TITLE
For the "Kiwix App" section, consider having a "Progressive Web App" category in dropdown

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/web#223]
@Popolechien This is a suggested solution for the dilemma raised in #217. I suggest having a new entry in the dropdown on the download page. This could be "Progressive Web App" (or "PWA" if latter is too long).

This would allow having a dedicated tile for the PWA, explaining what it is compatible with and where it can be installed. If you like the idea, I can provide some suggested text.

NB This would not supersede the UWP and Electron version listed on the Windows tile, as those are installable packages from app stores or GitHub releases, very different backend to the PWA.

**Comment by Popolechien** [comment]
Yep, let's do it, though as indicated in #221 there may be some display issues if users try to use the filters, but then I'm not sure it's a majority of users that use these, the page isn't that long to begin with. 

Does this mean that we'll need to remove all PWA buttons from the existing download CTAs?

**Comment by kelson42** [comment]
@Popolechien I don't really understand, because we have discussed twice these topics in meetings - with decision. So why the discussion is still going on?

**Comment by Jaifroid** [comment]
I don't know anything about your discussions or what was decided, but to answer @Popolechien, I'd say if there is a dedicated entry in the dropdown, it would mean you could remove any separate PWA buttons on other tiles (but not the Electron buttons).

@kelson42 If you don't like the idea, then I'd say there should be a PWA button on tiles for relevant OS's where the PWA can be installed (pretty much universal).

As Google is about to publish a dedicated piece on the PWA, it looks strange if Kiwix itself doesn't know how to present it on its own app pages!

If you have concerns about the PWA, please let me know what they are. I'm just trying to suggest solutions to the lack of fit between an OS-centric display of apps on the Kiwix download page, and the fact that PWAs are universal by design and don't fit into OS boxes.
